tests/torch/test_pytorch_patch.py

```diff
@@ -0,0 +1,28 @@
+from nncf.torch.dynamic_graph.patch_pytorch import MagicFunctionsToPatch
+from nncf.torch.graph.operator_metatypes import PT_OPERATOR_METATYPES
+
+
+def test_is_any_metatype_op_names_duplicated():
```

Comment:
The test name should reflect what it asserts.
`test_metatype_op_names_do_not_intersect`

Reply:
Done